src/socket-proxy/socket-proxyd.c

```diff
@@ -426,6 +449,12 @@ static int add_connection_socket(Context *context, int fd) {
                 return 0;
         }
 
+        r = sd_event_source_set_enabled(context->idle_time, SD_EVENT_OFF);
+        if (r < 0) {
+                log_error_errno(r, "Unable to disable idle timer: %m");
```

Comment:
It doesn't matter too much, since this is unlikely to fail, but please downgrade this to warning (since it's not fatal), and add ", ignoring" to tell the reader that we intend to continue, and remove braces around a single line. I.e. something like that:
```C
if (r < 0)
     log_warning_errno(r, "Unable to disable idle timer, continuing: %m");
```